packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-add-step.ts

```diff
@@ -69,7 +69,7 @@ export interface EmrAddStepProps {
    *
    * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_HadoopJarStepConfig.html
    *
-   * @default No mainClass
+   * @default - No mainClass
    */
   readonly mainClass?: string;
```

Comment:
it should be optional. You're right that a main class is required. It can be supplied directly using this parameter or specified in the manifest file of a supplied `jar`